app/change_set_persisters/plum_change_set_persister.rb

```diff
@@ -56,6 +56,7 @@ def before_save(change_set:)
 
     def after_save(change_set:, updated_resource:)
       append(append_id: change_set.append_id, updated_resource: updated_resource) if change_set.append_id.present?
+      IdentifierService.mint_or_update(updated_resource) if change_set.try(:state_changed?) && change_set.new_state == 'complete'
```

Comment:
Oh, I don't think this works. I don't think it ever gets called. There's only one spec, which sets a source_metadata_identifier, so it's being called as part of refreshing remote metadata. Part of why I don't like postfix conditionals - SimpleCov would have caught this